Lombiq.BaseTheme/Services/ICssClassHolder.cs

```diff
@@ -22,7 +28,14 @@ public interface ICssClassHolder
     /// Returns the set of classes belonging to the zone called <paramref name="zoneName"/>. You can use this to
     /// remove classes if needed.
     /// </summary>
+    [Obsolete($"Use {nameof(GetOrAddZoneClasses)} instead.")]
     ISet<string> GetZoneClasses(string zoneName);
+
+    /// <summary>
+    /// Returns the set of classes belonging to the zone called <paramref name="zoneName"/>. If the set doesn't exists,
+    /// a new one is created. You can use this to remove classes if needed.
+    /// </summary>
+    ISet<string> GetOrAddZoneClasses(string zoneName);
```

Comment:
I think the name does not correctly reflect the method's intent, because we're not adding any zone classes here. If we don't want to invest more time into this, I would just keep the old name and skip the obsoletion. I don't think we're going to break anyone's code.

In a world of beautiful APIs, I would prefer to write:

```c#
CssClassHolder.Zone(zoneName).Remove(someClass).Add(anotherClass).Add(oneMore);
```

But that would entail writing more code for a feature we are using in only a very few places.

Reply:
It's normal to use GetOrAdd instead of Get for self-initializing access, like in ConcurrentDictionary or the extension method code I've linked in the issue. I think it's pretty clear what we are adding from the method's signature.

Reply:
If the method name was simply `GetOrAdd`, that would be fine. But we're carrying the "ZoneClasses" prefix around, and there are no zone classes being passed to this method. That **is** confusing to me, and differs from [ConcurrentDictionary.GetOrAdd(key, value)](https://learn.microsoft.com/en-us/dotnet/api/system.collections.concurrent.concurrentdictionary-2.getoradd#system-collections-concurrent-concurrentdictionary-2-getoradd(-0-1)).

If this wasn't a versioned, public API, I wouldn't care much, because we could always rename it later. But in this case, it means extra effort, so I'd like to get the name straight before publishing it.